tilecloud_chain/internal_mapcache.py

```diff
@@ -222,6 +223,11 @@ def fetch(server, tilegeneration, layer, tile, kwargs):
                 generator.compute_tile(meta_tile)
 
                 fetched_tile = generator.read_from_cache(tile)
+
+                if fetched_tile is None:
+                    time.sleep(0.1)
```

Comment:
when do you need this? Isn't `read_from_cache` a blocking request? 100ms for a tile seems uge

Reply:
Updated, return the created tile